rendermodules/mesh_lens_correction/do_mesh_lens_correction.py

```diff
@@ -143,46 +155,47 @@ def run(self):
         # generate tile pairs for this section in the input stack
         tp_example = self.generate_tilepair_example()
         tp_mod = TilePairClientModule(input_data=tp_example,
-                                        args=['--output_json', out_file.name])
+                                      args=['--output_json', out_file.name])
         tp_mod.run()
 
         with open(tp_mod.args['output_json'], 'r') as f:
             js = json.load(f)
         self.args['pairJson'] = js['tile_pair_file']
 
-        # generate point matches
-        generate_point_matches(self.render,
-                               self.args['pairJson'],
-                               self.args['input_stack'],
-                               self.args['match_collection'],
-                               self.args['matchMax'],
-                               nfeature_limit=self.args['nfeature_limit'])
-
-        meshclass = MeshAndSolveTransform()
+        if self.args['rerun_pointmatch']:
```

Comment:
shouldn't the existing point matches be deleted by default if they exists?. This will make the get_tilespecs_and_matches function in MeshAndSolve to fail if the tile_ids that exist in point matches don't exist in the stack.

The get_tilespecs_and_matches is getting its "unique_ids" from the match collection and not the tilespecs.

Reply:
for debugging purposes, especially when just working with the mesher/solver/transform maker, it is nice to not have to rerun the pointmatches.
it also helps with test coverage without having to rerun the pointmatches in every case.
the default is to delete them.

Reply:
also, the get_tilespecs_and_matches is getting unique ids from the intersection of stacks and matches.

Reply:
ok makes sense